packages/core/lib/send-result/node-http.ts

```diff
@@ -42,9 +42,15 @@ export async function sendMultipartResponseResult(
   // @ts-expect-error - Different Signature between ServerResponse and Http2ServerResponse but still compatible.
   rawResponse.write("---");
```

Comment:
I usually prefer to do "as X" to resolve ts-errors, just because it workarounds the actual issue and not the entire line... which might have other errors